mlir/python/mlir/dialects/transform/structured.py

```diff
@@ -373,10 +373,11 @@ class PadOp(PadOp):
     def __init__(
         self,
         target: Union[Operation, OpView, Value],
+        pad_to_multiple_of: Optional[Union[DynamicIndexList, ArrayAttr]] = None,
```

Comment:
I'm not sure what happens with typing here. Ideally, we should take a list where one can mix python integers, handles and attributes freely OR an array attribute.

Reply:
to be honest, i pretty much copy pasta'd from vectorize op for this. i might need to think about this to understand

Reply:
Just check out the definition of `DynamicIndexList`. It seems to be more or less what I described.

Reply:
yes. so i believe this typing is correct then?